Client/Ecosia/UI/WhatsNew/WhatsNewViewModel.swift

```diff
@@ -0,0 +1,18 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+final class WhatsNewViewModel {
+    var items: [WhatsNewItem]
+    
+    init(provider: WhatsNewDataProvider) {
+        do {
+            self.items = try provider.getData()
+        } catch {
+            print("Failed to fetch Whats new items. Reason \(error)")
+            self.items = []
+        }
```

Comment:
I guess with the local there is no chance of error anyway... Just a simplification suggestion, let me knwo what you think 🙂 

(Thought about adding a todo to handle error when they actually happen, but don't think it is that relevant, leaving it up to you)

```suggestion
        self.items = try? provider.getData() ?? []
```

Reply:
Totally up -> hence committing 😝. Thanks!